packages/cozy-procedures/src/components/Overview.jsx

```diff
@@ -1,36 +1,90 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Link, withRouter } from 'react-router'
+import { withRouter } from 'react-router'
+import Topbar from './Topbar'
 import creditApplicationTemplate from '../templates/creditApplicationTemplate'
+import {
+  Title,
+  SubTitle,
+  Caption,
+  Button,
+  translate
+} from 'cozy-ui/transpiled/react'
 
-const Overview = ({ personalData, location }) => {
-  const personalDataFieldsTotal = Object.keys(personalData).length
-  const personalDataFieldsCompleted = Object.values(personalData).filter(
-    Boolean
-  ).length
+class Overview extends React.Component {
+  navigateTo = view => {
+    const { location, router } = this.props
+    const rootPath = location.pathname
+    const separator = rootPath.endsWith('/') ? '' : '/'
+    router.push(`${rootPath}${separator}${view}`)
+  }
 
-  const rootPath =
-    location.pathname + (location.pathname.endsWith('/') ? '' : '/')
+  render() {
+    const { personalData, t } = this.props
 
-  return (
-    <div>
-      <h1>{creditApplicationTemplate.name}</h1>
-      <div>
-        <Link to={`${rootPath}other`}>Other</Link>
-      </div>
+    const personalDataFieldsTotal = Object.keys(personalData).length
```

Comment:
nit: I will probably move this in a separate function to be able to use it in the form.